src/www/interfaces.php

```diff
@@ -2280,7 +2280,7 @@ function toggle_wirelesscfg() {
                           <td>
                             <input name="dhcprejectfrom" type="text" id="dhcprejectfrom" value="<?=htmlspecialchars($pconfig['dhcprejectfrom']);?>" />
                             <div class="hidden" data-for="help_for_dhcprejectfrom">
-                              <?=gettext("If there is a certain upstream DHCP server that should be ignored, place the IP address or subnet of the DHCP server to be ignored here."); ?>
+                              <?=gettext("If there are certain upstream DHCP servers that should be ignored, place the comma separated list of one or more IP addresses of the DHCP server(s) to be ignored here."); ?>
```

Comment:
A bit to uncertain about itself here with conditionals and "(s)" use. It's probably better to rephrase this in a snappier way.

Reply:
Simplified